aws/resource_aws_s3_bucket_test.go

```diff
@@ -1484,15 +1517,8 @@ EOF
 
 func testAccAWSS3BucketConfigWithAcceleration(randInt int) string {
 	return fmt.Sprintf(`
-provider "aws" {
-	alias = "west"
-	region = "eu-west-1"
```

Comment:
It looks like `testAccAWSS3BucketConfigWithoutAcceleration` needs to be updated as well

```
=== RUN   TestAccAWSS3Bucket_acceleration
--- FAIL: TestAccAWSS3Bucket_acceleration (243.29s)
    testing.go:518: Step 1 error: Error refreshing: 1 error(s) occurred:
        
        * aws_s3_bucket.bucket: 1 error(s) occurred:
        
        * aws_s3_bucket.bucket: aws_s3_bucket.bucket: error reading S3 bucket "tf-test-bucket-9115369584085629874": BucketRegionError: incorrect region, the bucket is not in 'eu-west-1' region
```